html/apps/signup/css/style.css

```diff
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: 'Lato';
+  font-style: normal;
+  font-weight: 400;
+  src: local('Lato Regular'), local('Lato-Regular'), url(http://themes.googleusercontent.com/static/fonts/lato/v7/qIIYRU-oROkIk8vfvxw6QvesZW2xOQ-xsNqO47m55DA.woff) format('woff');
+}
+
+body {
+	background: #448ed3 ;
+	font-family: "Lato" ;
+}
+img{
+    width:15%;
+    height: 15%
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected end of value but found 'max-width'.](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2589955281/issues/source?bid=3288751&fileBranchId=3305649#l14)